cmd/vic-machine/configure/configure.go

```diff
@@ -224,21 +221,14 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 	if n.CertificateAuthorities != nil {
 		o.CertificateAuthorities = n.CertificateAuthorities
 	}
+
 	if n.UserCertificates != nil {
 		o.UserCertificates = n.UserCertificates
 	}
 
 	if n.RegistryCertificateAuthorities != nil {
```

Comment:
@emlin these three lines copy across `c.RegistryCAs` (they get copied to `n` in `Validate`) to the new config

I included these three lines mistakenly as a part of `configure certs`, but they're a no-op as-is since nothing ever sets those value. After this PR is merged they will no longer be a no-op.